debug/.bashrc

```diff
@@ -0,0 +1,114 @@
+# ~/.bashrc: executed by bash(1) for non-login shells.
+# see /usr/share/doc/bash/examples/startup-files (in the package bash-doc)
+# for examples
+
+# If not running interactively, don't do anything
+case $- in
+    *i*) ;;
+      *) return;;
+esac
+
+# don't put duplicate lines or lines starting with space in the history.
+# See bash(1) for more options
+HISTCONTROL=ignoreboth
+
+# append to the history file, don't overwrite it
+shopt -s histappend
+
+# for setting history length see HISTSIZE and HISTFILESIZE in bash(1)
+HISTSIZE=1000
```

Comment:
I know this is probably a default system bashrc but could be useful to set the history higher, `1000` is really small!

Reply:
Ha, good point. I took my `.bashrc` as it was. I don't use bash, so I haven't tweaked it at all.

I'll bump this to 100k.

If you have any other suggestions or requests about the `.bashrc` those would be very welcome. The goal of this image is to provide comfortable working environment for people, so they should feel at home.

Examples:
 - install zsh/fish + config?
 - install emacs + config?